packages/webpack2/__tests__/integration.test.js

```diff
@@ -23,37 +23,37 @@ test('complete webpack config creation', (t) => {
   ])
 
   t.is(webpackConfig.module.loaders.length, 8)
-  t.deepEqual(webpackConfig.module.loaders[0], {
```

Comment:
Ahh, if the order has changed here, we should re-order the assertions as well. Having the indexes in the order 7, 6, 0, 1, 2 really is confusing.